catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -64,7 +74,13 @@ const SearchForm = ({
   // Router
   const [inputQuery, setInputQuery] = useState(query);
   const [enhanced, setEnhanced] = useState(false);
-  const searchInput = useRef(null);
+  const searchInput = useRef();
+  const submit = () => {
```

Comment:
The idea here is that we never use a function to update URLs etc, but rather just submit the form and let the standard behaviour we have attached.